src/packageurl/__init__.py

```diff
@@ -72,7 +72,7 @@ def unquote(s: AnyStr) -> str:
     Return a percent-decoded unicode string, given an `s` byte or unicode
     string.
     """
-    unquoted = _percent_unquote(s)  # type:ignore[arg-type]  # typeshed is incorrect here
```

Comment:
Are you sure this will not make typeshed or mypy balk here?

Reply:
@pombredanne this change was done due to mypy error only, mypy tests are passing with this change.